domain/src/main/java/com/github/dodobest/domain/InputHandler.kt

```diff
@@ -19,7 +19,7 @@ class InputHandler {
 
     fun handleInputNum (inputNum: String) {
         if (arraySize > 0 && expression.isNum(numAndSignArray.last().toCharArray().last())) {
-            numAndSignArray[arraySize-1] = (numAndSignArray.last().toInt() * 10 + inputNum.toInt()).toString()
+            numAndSignArray[arraySize-1] = (numAndSignArray.last().toDouble().toInt() * 10 + inputNum.toInt()).toString()
```

Comment:
```suggestion
            numAndSignArray[numAndSignArray.lastIndex] = (numAndSignArray.last().toDouble().toInt() * 10 + inputNum.toInt()).toString()
```
lastIndex를 활용하면 좋을 것 같아요.

Reply:
10을 곱하는 이유는 무엇일까요?

매직 넘버란 코드 안에 작성된 구체적인 숫자 혹은 문자열을 말하며, 매직 넘버의 단점은 아래와 같습니다.
- 의미를 이해하기 어렵다.
- 복수의 장소에서 사용된다면 변경하기 힘들다.

그런 매직 넘버는 의미를 나타낼 수 있는 상수로 치환하여 코드의 가독성을 높여 보는 것은 어떨까요?